core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/deploy/maven/MavenCentralMavenDeployer.java

```diff
@@ -191,7 +191,7 @@ public Integer getReadTimeout() {
     };
 
     public MavenCentralMavenDeployer() {
-        super(org.jreleaser.model.api.deploy.maven.MavenCentralMavenDeployer.TYPE);
+        super(org.jreleaser.model.api.deploy.maven.MavenCentralMavenDeployer.TYPE, true);
```

Comment:
Oops I made a mistake and this deployer _does_ not support snapshots at all. Please roll back to previous.

Reply:
Doesn't it ? I mean, it's technically possible to deploy snapshots to mavenCentral, right? But it repo separated or something? 

Can we [do the equivalent of ](https://central.sonatype.org/publish/publish-guide/#releasing-to-central):

```
OSSRH Usage Notes[⚓︎](https://central.sonatype.org/publish/publish-guide/#ossrh-usage-notes)

Accessing Repositories[⚓︎](https://central.sonatype.org/publish/publish-guide/#accessing-repositories)

The following repositories allow you to access your components in OSSRH directly. Users will simply get your components directly via the Central Repository, but for committers and other collaborators on your project this list might be useful.

Repository URL for snapshot deployment and download access:

https://s01.oss.sonatype.org/content/repositories/snapshots/
Repository URL for release deployment, no download access! :

https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/
```

With JRelaser for maven central/sonatype, and in this case, how do we do it?